gnssanalysis/_version.py

```diff
@@ -138,13 +154,13 @@ def versions_from_parentdir(parentdir_prefix, root, verbose):
 
 
 @register_vcs_handler("git", "get_keywords")
-def git_get_keywords(versionfile_abs):
+def git_get_keywords(versionfile_abs: str) -> Dict[str, str]:
     """Extract version information from the given file."""
```

Comment:
One-line sphinx docstring (missing the description of input parameters, what's returned, etc.: `:param` `:return`)

Reply:
file autogenerated by the versionner script. so won't be addressed